src/main/kotlin/com/avail/optimizer/RegisterSet.kt

```diff
@@ -0,0 +1,700 @@
+/*
+ * RegisterSet.java
```

Comment:
`java` → `kt`

Reply:
Please check that all of the copyrights in this port have correctly named files.